ScreenConfig.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+import "brewblox.proto";
+import "nanopb.proto";
+
+package blox.ScreenConfig;
+
+enum Type {
+  Row = 0;
+  Column = 1;
+  Content = 2;
+}
+
+message LayoutNode {
+  uint32 parent = 1 [(nanopb).int_size = IS_8];
+  uint32 nodeId = 2 [(nanopb).int_size = IS_8];
+  Type type = 3;
+  uint32 weight = 4 [(nanopb).int_size = IS_16];
+}
+
+message Color {
+  uint32 r = 1 [(nanopb).int_size = IS_8];
+  uint32 g = 2 [(nanopb).int_size = IS_8];
+  uint32 b = 3 [(nanopb).int_size = IS_8];
+}
+
+message NumericValueWidget {
+  Color color = 1;
+  uint32 value = 2 [(nanopb).int_size = IS_8];
+  string label = 3 [(nanopb).max_length = 40];
+}
+
+message ColorWidget {
+  Color color = 1;
+}
+
+message ContentNode {
+  uint32 layoutNodeId = 1 [(nanopb).int_size = IS_8];
+  oneof Content {
```

Comment:
`Content` -> `content`

Reply:
fixed in f18dc1a6cdfbbd4f489edf5f8b2de641368e5911